test/functional/test-url.js

```diff
@@ -429,6 +430,10 @@ describe('isProxyOrigin', () => {
   testProxyOrigin(
       'http://cdn.ampproject.org/', false);
   testProxyOrigin(
+    'https://cdn.ampproject.org.badguys.com/', false);
```

Comment:
Please add this case `https://cdn.ampproject.orgbadguys.com/`

Reply:
Done